TBB related compilation error in DDDigi

```
[ 72%] Building CXX object DDDigi/CMakeFiles/DDDigi.dir/src/DigiKernel.cpp.o
In file included from /home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:27:
/usr/include/tbb/tbb.h:21:154: note: #pragma message: TBB Warning: tbb.h contains deprecated functionality. For details, please see Deprecated Features appendix in the TBB reference manual.
   21 | ee Deprecated Features appendix in the TBB reference manual.")
      |                                                              ^

/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp: In member function 'virtual void dd4hep::digi::DigiKernel::submit(const dd4hep::digi::DigiAction::Actors<dd4hep::digi::DigiEventAction>&, dd4hep::digi::DigiContext&) const':
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:276:34: error: no matching function for call to 'dd4hep::digi::DigiKernel::Wrapper::Wrapper(dd4hep::digi::DigiContext&, dd4hep::digi::DigiEventAction&)'
  276 |       que.run(Wrapper(context, *i));
      |                                  ^
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:95:3: note: candidate: 'constexpr dd4hep::digi::DigiKernel::Wrapper::Wrapper(const dd4hep::digi::DigiKernel::Wrapper&)'
   95 |   Wrapper(const Wrapper& copy) = default;
      |   ^~~~~~~
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:95:3: note:   candidate expects 1 argument, 2 provided
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:94:3: note: candidate: 'constexpr dd4hep::digi::DigiKernel::Wrapper::Wrapper(dd4hep::digi::DigiKernel::Wrapper&&)'
   94 |   Wrapper(Wrapper&& copy) = default;
      |   ^~~~~~~
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:94:3: note:   candidate expects 1 argument, 2 provided
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:92:3: note: candidate: 'dd4hep::digi::DigiKernel::Wrapper::Wrapper(dd4hep::digi::DigiContext&, dd4hep::digi::DigiEventAction*)'
   92 |   Wrapper(DigiContext& c, DigiEventAction* a)
      |   ^~~~~~~
/home/arch/DD4hep/DDDigi/src/DigiKernel.cpp:92:44: note:   no known conversion for argument 2 from 'dd4hep::digi::DigiEventAction' to 'dd4hep::digi::DigiEventAction*'
   92 |   Wrapper(DigiContext& c, DigiEventAction* a)
      |                           ~~~~~~~~~~~~~~~~~^
make[2]: *** [DDDigi/CMakeFiles/DDDigi.dir/build.make:200: DDDigi/CMakeFiles/DDDigi.dir/src/DigiKernel.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2035: DDDigi/CMakeFiles/DDDigi.dir/all] Error 2
make: *** [Makefile:161: all] Error 2
```
OS: arch linux
compiler: gcc 9.3
cmake: 3.17
tbb: 2020/2

--- a/DDDigi/src/DigiKernel.cpp
+++ b/DDDigi/src/DigiKernel.cpp
@@ -273,7 +273,7 @@ void DigiKernel::submit(const DigiAction::Actors<DigiEventAction>& actions, Digi
   if ( parallel )   {
     tbb::task_group que;
     for ( auto* i : actions )
-      que.run(Wrapper(context, *i));
+      que.run(Wrapper(context, i));
     que.wait();
     goto print_stamp;
   }

